internal/snap_printer/snap_printer_tracking.go

```diff
@@ -112,7 +112,7 @@ func (p *printer) prependTopLevelDecls() {
 			}
 			decl += v.name
 		}
-		decl += ";\n"
```

Comment:
I assume this is change is the reflect I am seeing in the "looks like this could have been left" comment above?

Reply:
We talked about this offline, but basically in order to fix sourcemaps for now, I had to mess with the formatting of the generated code. The generated code is functional and properly sourcemapped but just doesn't look 100% formatted correctly. I logged this to address this later: https://github.com/cypress-io/esbuild/issues/11